pkg/app/gen/html.go

```diff
@@ -1284,6 +1284,11 @@ var attrs = map[string]attr{
 	},
 
 	// K:
+	"key-value": {
+		Name: "SetAttribute",
+		Type: "key|value",
```

Comment:
`key|value` is a pretty generic to call a pair while here, we are explicitly dealing with an attr.
Let's call that `attr|value`.